package.json

```diff
@@ -6,21 +6,38 @@
   "author": "Matan Kushner <matchai@me.com>",
   "scripts": {
     "test": "fish tests/run.fish",
-    "version": "scripts/version.fish",
     "docs:prepare": "gitbook install",
-    "docs:serve": "npm run docs:prepare && gitbook serve .",
     "docs:build": "npm run docs:prepare && gitbook build",
+    "docs:serve": "npm run docs:prepare && gitbook serve .",
     "contributors:add": "all-contributors add",
     "contributors:generate": "all-contributors generate"
   },
   "license": "MIT",
   "devDependencies": {
+    "@semantic-release/exec": "^3.3.1",
+    "@semantic-release/git": "^7.0.6",
     "all-contributors-cli": "^5.4.1",
-    "gitbook-cli": "^2.3.2",
```

Comment:
It appears these gitbook dependencies were not needed in the first place, and are downloaded by gitbook when `gitbook install` is run.